ReactApp/src/utils/rest-api-call.js

```diff
@@ -1,4 +1,5 @@
 import Cookies from 'js-cookie';
+import {validateParameters} from '../../../src/main/validateParameters'
```

Comment:
does this work even though this file is completely out of scope?

Reply:
Yes, seems like it ,as far as i have tested...